src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementService.java

```diff
@@ -86,6 +86,21 @@ public ConfirmationStatementService(CompanyProfileService companyProfileService,
         }
     }
 
+    public ResponseEntity<Object> getConfirmationStatement(String submissionId) {
```

Comment:
not sure if the service should return a responseEntity, i'd leave that to the controller. I think this method should just return the confirmation statement found in the repository.